BSU-SUITS-Team/ARSIS-6
# Lmcc updates

# Description

This adds a mock store that can be used for testing and displays the store on the biometrics table page

Closes #200

## Bug Fix or Feature

Feature/Work in progress

# Testing Instructions

- Open LMCC
- Navigate to `/biometrics`
   - Verify that the are changing values

## Linting

- [x] I've linted my code
